lib/taskana-core/src/main/java/pro/taskana/spi/history/api/events/task/ClaimCancelledEvent.java

```diff
@@ -6,7 +6,7 @@
 public class ClaimCancelledEvent extends TaskEvent {
 
   public ClaimCancelledEvent(Task task, String userId) {
-    super(task, userId);
+    super(task, userId,"");
```

Comment:
should we rather use 'null'? I just don't want that the database created an empty CLOB for an empty String. Do you know how it is handled?

Reply:
Was considering null too, changed this now